src/pages/standards/colour.js

```diff
@@ -91,11 +91,11 @@ export default () => (
           />
           <Subheading>Focus</Subheading>
           <Swatches
-            swatches={[[{ label: "Yellow", hex: "#ffbf47", primary: true }]]}
+            swatches={[[{ label: "Orange", hex: "#e86c00", primary: true }]]}
```

Comment:
@ritapatel7, I am using this hex value for 'Orange' #ba5600 to make it pass accessibility. I have also change 'Teal' to this hex value: #007a87